demos/chips.html

```diff
@@ -54,18 +54,82 @@
       </section>
 
       <section class="example">
+        <h2>Entry Chips</h2>
         <div class="mdc-chip-set">
-          <div class="mdc-chip">
-            <div class="mdc-chip__text">Add to Calendar</div>
+          <div class="demo-chip mdc-chip mdc-chip--with-leading-icon mdc-chip--with-trailing-icon">
+            <i class="material-icons mdc-chip__icon" tabindex="0">face</i>
+            <div class="mdc-chip__text">Jane Smith</div>
+            <i class="material-icons mdc-chip__icon mdc-chip__icon--trailing" tabindex="0">more_vert</i>
```

Comment:
Add ARIA and title attributes: `role="button" title="More options"`

Ditto for other elements that have a `tabindex="0"`

Reply:
Should chips themselves also have `role="button"`? I guess this is related to [Ken's comment](https://github.com/material-components/material-components-web/pull/2191/files/9a3b9467d3ba4e4ec9e28713a177079566f3af16#r167957686) above.

Reply:
I suspect that once all the variants of chips are in, we should document guidance on what roles make sense for what types of chips.